### ISSUE TITLE
Accessibility: label mandatory fields

### ISSUE THREAD

**Issue by michelefedele** [SondagesPro/LS-theme-skelvanilla#1]
Ciao, 
I am using your theme for a survey and i noticed an accessibility problem when there are mandatory fields on the page. In your theme you have entered the string "required" visible only by screen readers. To be accessible, however, the string should be visible to all the users while the screen reader should be alerted by an aria-required or one of the other techniques listed here:
https://www.w3.org/TR/2010/NOTE-UNDERSTANDING-WCAG20-20101014/minimize-error-cues.html

Is it possible to solve this problem? 
Thanks
Michele

**Comment by Shnoulle** [comment]
https://gitlab.com/SondagesPro/SurveyThemes/skelvanilla/-/issues/45
